ipaserver/install/server/__init__.py

```diff
@@ -595,6 +595,8 @@ class ServerReplicaInstall(ServerReplicaInstallInterface):
     subject_base = None
     ca_subject = None
 
+    force_join = client.ClientInstallInterface.force_join
```

Comment:
I meant this bit as well, `force_join` is already present in `ServerReplicaInstall` through inheritance.